code/game/machinery/Sleeper.dm

```diff
@@ -279,3 +279,25 @@
 
 /obj/machinery/sleeper/old
 	icon_state = "oldpod"
+
+/obj/machinery/sleeper/party
+	name = "party pod"
+	desc = "'Sleeper' units were once known for their healing properties, until a lengthy investigation revealed they were also dosing patients with deadly lead acetate. This appears to be an old 'sleeper' unit repurposed as a 'Party Pod'. It’s probably not a good idea to use it."
+	icon_state = "partypod"
+	idle_power_usage = 3000
+	circuit = /obj/item/circuitboard/machine/sleeper/party
+	ui_x = 260
+	ui_y = 375
+
+	controls_inside = TRUE
+	possible_chems = list(
+		list(/datum/reagent/consumable/ethanol/beer, /datum/reagent/consumable/laughter),
+		list(/datum/reagent/spraytan),
+		list(/datum/reagent/colorful_reagent),
+		list(/datum/reagent/drug/space_drugs)
+	)//Exclusively uses non-lethal, "fun" chems. At an obvious downside.
+	enter_message = "<span class='notice'><b>You're surrounded by some funky music inside the chamber. You go deaf as you feel waves of krunk vibe within you.</b></span>"
+
+/obj/machinery/sleeper/party/inject_chem(chem, mob/user)
+	. = ..()
+	occupant.reagents.add_reagent(/datum/reagent/toxin/leadacetate, 4) //You're injecting chemicals into yourself from a recalled, decrepit medical machine. What did you expect?

```

Comment:
Maybe only make this (and old sleepers) inject you with lead acetate if emagged? Then the L O R E behind the sleeper recall would be that the Syndicate rigged the quality testing so that Nanotrasen would recall some of their most effective medical machinery.

Reply:
On the one hand, muh consistency. On the other hand, having the old sleepers that are present in some ruins inject you with lead acetate would be annoying as fuck gameplay wise and make them literally cancerous to use, hence my emag effect suggestion.

Reply:
I’d say make it a chance instead of guaranteed and a random amount from 1-4

Reply:
I'd also like some of the more harmful meme chemical options on emag Ig: bone hurting juice, itching powder, mint toxin, mime's bane, and spewium.

Reply:
Yeah i think it shouldn't normally cause any damage, so that the clown is able to shove you in the sleeper and load you up for maximum prankage.

Reply:
I think what I'll do to balance that edge between "overtly dangerous" and "mildly useful" is throw the chance of injecting lead to like 20% usually, _Unless_ it's been emagged, in which case full dosage all the way.